src/Features/OnboardingTasks/TaskLists.php

```diff
@@ -88,255 +96,33 @@ public static function add_defaults() {
 		);
 
 		self::add_task( 'setup', StoreDetails::get_task() );
+		self::add_task( 'setup', Purchase::get_task() );
+		self::add_task( 'setup', Products::get_task() );
+		self::add_task( 'setup', WooCommercePayments::get_task() );
+		self::add_task( 'setup', Payments::get_task() );
+		self::add_task( 'setup', Tax::get_task() );
+		self::add_task( 'setup', Shipping::get_task() );
+		self::add_task( 'setup', Marketing::get_task() );
```

Comment:
The appearance task is missing here.

Reply:
😱  Must have been distracted with the releases yesterday.  Thanks for catching this!